src/components/CarCard.tsx

```diff
@@ -1,65 +1,72 @@
 import React from "react";
 import { ReactComponent as CalendarIcon } from "@assets/fa-calendar.svg";
-import { ReactComponent as TruckIcon } from "@assets/fa-truck.svg";
-import { ReactComponent as CogIcon } from "@assets/fa-cog.svg";
-import { ReactComponent as HammerIcon } from "@assets/fa-hammer.svg";
-import { ReactComponent as ToolboxIcon } from "@assets/fa-toolbox.svg";
-import { ReactComponent as ScrewdriverIcon } from "@assets/fa-screwdriver.svg";
+import { ReactComponent as MitOstIcon } from "@assets/MitOst.svg";
+import { ReactComponent as CadusIcon } from "@assets/Cadus.svg";
+import { ReactComponent as HIWWIcon } from "@assets/HIWW.svg";
+import { ReactComponent as OstrivIcon } from "@assets/Ostriv.svg";
+import type { ITolocarsFrontmatter } from "@interfaces/ITolocars";
+import { Tag } from "@components";
 
-interface Props {
+interface Props extends ITolocarsFrontmatter {
   className?: string;
-  title: string;
-  subtitle: string;
-  tag: string;
-  date: string;
-  operator: string;
-  img: string;
-  tagIcon?: "truck" | "hammer" | "cog" | "toolbox" | "screwdriver";
+  href?: string;
 }
 
 const CarCard: React.FC<Props> = ({
   className,
+  href,
   img,
   title,
-  subtitle,
-  tag,
+  tags,
   date,
-  tagIcon,
+  operatorIcon,
```

Comment:
Can we just name this `operators`? They're not really icons, but logos in my opinion. Also what happens if we have multiple operators (there's one of these examples in the Figma design)? I'd make this an array.